src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -1462,6 +1460,12 @@ def set_columns(self, new_pandas_labels: Axes) -> "SnowflakeQueryCompiler":
         )
         return SnowflakeQueryCompiler(new_internal_frame)
 
+    # TODO SNOW-837664: add more tests for df.columns
```

Comment:
Minor nit, but should we separate this out into two methods? i.e. def _get_columns(self), def _set_columns(self, labels), columns = property(_get_columns, _set_columns)? It's more verbose, but looks nicer, and is consistent with the rest of the codebase, but again, bot a blocker.

Reply:
sorry, where do we have _set_columns, I only see set_columns. why do we need so many set_columns? And i agree the function name should be consistent, either all with _, or both not

Reply:
It was initially `_set_columns` when Rehan first reviewed it. I've removed that method now, and it's just `get_columns` (newly added) and `set_columns` (existing) referenced in the property.